snippets/base/models.py

```diff
@@ -1401,6 +1440,7 @@ class ASRSnippet(models.Model):
             'See the current time in <a target="_blank" href="http://time.is/UTC">UTC</a>'))
 
     locales = models.ManyToManyField('TargetedLocale', blank=True, verbose_name='Targeted Locales')
```

Comment:
Do we still need `locales`?

Reply:
Yes, for now. I want to run a data migration script manually after this reaches production and then I'll pull request a migration to remove `locales`.